src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,96 @@
+.contacts {
+  &__wrapper {
+    @include pageGrid;
+    grid-template-rows: repeat(4, 56 248px 56px 244px);
+
+    .section-title {
+      grid-column: span 2;
+      grid-row: 1;
+
+      @include onTable {
+        grid-column: 3/6;
+      }
+      @include onDesktop {
+        grid-column: 5/9;
+      }
+    }
+
+    .form {
+      grid-column: span 2;
+      grid-row: 2/4;
+
+      @include onTable {
+        grid-column: span 3;
+      }
+      @include onDesktop {
+        grid-column: span 5;
+        grid-row: 2/4;
+      }
+
+      &__input:hover,
+      &__input:focus,
+      &__textarea:hover,
+      &__textarea:focus {
+        color: $main-text-color;
+      }
+    }
+
+    .button {
+      grid-column: span 2;
+      grid-row: 4;
+      margin-bottom: 48px;
```

Comment:
Margin-bottom on this element might not have any effect because it is a grid item. Consider revising the positioning and margins of grid items.